tests/x11/xterm.pm

```diff
@@ -14,9 +14,15 @@ use base "x11test";
 use strict;
 use warnings;
 use testapi;
+use power_action_utils qw(power_action);
+use version_utils qw(is_leap);
 
 sub run {
     my ($self) = @_;
+    if (is_leap) {
+        power_action('reboot', textmode => 1);
+        $self->wait_boot(bootloader_time => 300);
+    }
```

Comment:
I suggest you add some description here: :)
record_info "the reason why you reboot the vm"